src/routes/quotes/data.json

```diff
@@ -1,25 +1,139 @@
 [
+  {
+    "background": "cais_statement",
+    "text": "Mitigating the risk of extinction from AI should be a global priority alongside other societal-scale risks such as pandemics and nuclear war.",
+    "author": "Statemen on AI risk,",
+    "author_description": "signed by hundreds of AI experts and other notable figures. Including 3 of the most cited scientists ever: Bengio, Hinton & Illya, and the 3 CEOs of the top 3 AI companies: Altman, Hassabis & Amodei.",
+    "color": "white",
+    "padding": "0rem"
+  },
+  {
+    "background": "hawking",
+    "text": "The development of full artificial intelligence could spell the end of the human race.",
+    "author": "Stephen Hawking",
+    "author_description": "Theoretical Physicist and Cosmologist",
+    "color": "white"
+  },
+  {
+    "background": "sunak",
+    "text": "Get this wrong, and AI could make it easier to build chemical or biological weapons. Terrorist groups could use AI to spread fear and destruction on an even greater scale. Criminals could exploit AI for cyber-attacks, disinformation, fraud, or even child sexual abuse. And in the most unlikely but extreme cases, there is even the risk that humanity could lose control of AI completely through the kind of AI sometimes referred to as ‘super intelligence’. Indeed, to quote the statement made earlier this year by hundreds of the world’s leading AI experts: “Mitigating the risk of extinction from AI should be a global priority alongside other societal-scale risks such as pandemics and nuclear war”.",
```

Comment:
Awesome quote!  But, might be a little to long to fit on these small cards?  
- Maybe add a [..] to drop a little from the middle somewhere?
- Or, try to use padding to shift it around?

<img width="337" alt="image" src="https://github.com/joepio/pauseai/assets/61096713/64358682-3b19-45fa-a110-a17927410cad">